idl2library.cmake

```diff
@@ -202,6 +202,7 @@ function(idl2library)
 
         # Group the IDL projects together
         set_target_properties(${current_idl_target} PROPERTIES FOLDER IDL)
+        set_target_properties(_opendds_codegen_1_for_${current_idl_target} PROPERTIES FOLDER IDL/generated)
```

Comment:
These targets are something that could and probably will change in the future, so they shouldn't be relied on in anyway. Going by the documentation of `FOLDER` are they showing up in your IDE?

Reply:
https://github.com/OpenDDS/OpenDDS/pull/5009

@doug1234 assuming we merge this in OpenDDS, do you still want to change OpenDDW?

Reply:
They are indeed showing up in the ide.  The OpenDDS change would fix the issue.  But I am not sure putting it in IDL/generated would be good in all cases.  For using idl2library it makes sense though.

Reply:
Just to be clear there are two levels of folders in what I am doing with idl2library.  The IDL folder contains the code for the libraries created from each idl file.  Under that is a generated folder that has all the dependency targets.